### PR TITLE
 modified CONTRIBUTING.md file by altering the grammar for better readability 

### PR DESCRIPTION
**Pull request by Mr-DG-Wick**
small grammatical changes to the CONTRIBUTING.md file

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ A blog post is available [on blog.js.cytoscape.org](http://blog.js.cytoscape.org
 
 If you'd like to contribute code to Cytoscape.js but you're not sure exactly what you'd like to implement, take a look at our [current milestones](https://github.com/cytoscape/cytoscape.js/milestones) to see what features we have planned in future --- or anything labelled [`help-wanted`](https://github.com/cytoscape/cytoscape.js/issues?q=is%3Aopen+is%3Aissue+label%3Ahelp-wanted).  Of course, we also welcome your own ideas.  You can discuss new ideas with the community on [GitHub discussions](https://github.com/cytoscape/cytoscape.js/discussions).
 
-Our goal is to make Cytoscape.js easy to use and comprehensive.  Thank you for taking the time and effort to contribute to help make that happen!
+Our goal is to make Cytoscape.js easy to use and comprehensive.  Thank you for taking the time and effort to contribute to Cytoscape.js.  It is your contribution that helps us acheive our goal.
```
Comment (maxkfranz) [inline .github/CONTRIBUTING.md:9]: The last sentence here doesn't read well.

**Review by maxkfranz — CHANGES_REQUESTED** [review]
Most of these changes look like they would improve understandability for contributors from non-English-speaking countries.  Great

Some minor edits are needed before this can be merged in.

Thanks for your feedback

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (Mr-DG-Wick) [reply .github/CONTRIBUTING.md:9]: thanks for the feedback!!
I'll modify it and commit again.

Comment (Mr-DG-Wick) [reply .github/CONTRIBUTING.md]: I modified the file as per your feedback . Kindly give it a look :)

**Comment by maxkfranz** [comment]
Thanks

**Pull request merged**